api/policy.go

```diff
@@ -32,13 +40,17 @@ const (
 	BackupPlanPolicyScheduleDaily BackupPlanPolicySchedule = "DAILY"
 	// BackupPlanPolicyScheduleWeekly is WEEKLY backup plan policy value.
 	BackupPlanPolicyScheduleWeekly BackupPlanPolicySchedule = "WEEKLY"
+
+	// AllowedWindowPeriodUnset is ANYTIME allowed window period cron.
+	AllowedWindowPeriodUnset AllowedWindowPeriodCron = "* * * * *"
```

Comment:
I guess empty string would be more appropriate than * * * * * to represent unset.

Reply:
Set this field to  * * * * * is out of the following consideration:
    1. keeping consistent with the cron format
    2. specifying the underlying meaning if user does not set this field: the environment does not have a window, and any ticket can be executed right after available